apps/web/src/app/[orgShortcode]/convo/layout.tsx

```diff
@@ -55,11 +54,67 @@ export default function Layout({
 }: Readonly<{ children: React.ReactNode }>) {
   const orgShortcode = useGlobalStore((state) => state.currentOrg.shortcode);
   const isMobile = useIsMobile();
+
+  const params = useParams();
+  const pathname = usePathname();
   const [showHidden, setShowHidden] = useState(false);
-  const { data: issueData, refetch: refetchIssues } =
-    platform.org.store.getOrgIssues.useQuery({
+
+  const isInConvo = !!params.convoId;
+  const isNewPage = pathname.endsWith('/convo/new');
+
+  function ConvoChildren() {
+    const { data: issueData } = platform.org.store.getOrgIssues.useQuery({
       orgShortcode
     });
```

Comment:
you should add a staleTime of 1 hour here so that it doesn't refetch on every navigation

Reply:
this is how it was
so you're asking for a new addition to this part?

Reply:
i want you to put a staleTime on the getOrgIssues query, otherwise its fetching it every time I change a convo or resize the window